File: verve patch/base/Applications/CSymphony_0_1/vbSamples/CodeEntryPoint.vb
Option Strict Off
Option Explicit On

Imports System
Imports System.CodeDom
Imports System.CodeDom.Compiler
Imports System.Collections
Imports CSymphony.Lib

Namespace CSymphony.CSy
	Public Class CSyUnit
		 Inherits GraphUnit
		Overrides Public Sub Build()
			ns("MyNameSpace")
			uses("System")

			Dim ConsoleWrite As MethodRef = New MethodRef ("System.Console",  "WriteLine")
			'Dim HelloWorld As Literal = New Literal("Hello World !!")

			declClass(TypePublic, "Class1")
			declEntry()
			enter (ex (ConsoleWrite, newLit("Hello World")))
		End Sub
	End Class
End Namespace
			

File: verve patch/base/Applications/CSymphony_0_1/csSamples/CodeEntryPoint.cs
using System;
using System.CodeDom;
using System.CodeDom.Compiler;
using System.Collections;
using CSymphony.Lib;

namespace CSymphony.CSy
{
	public class CSyUnit : GraphUnit
	{
		public override void Build()
		{
			ns("MyNameSpace");
			uses("System");

			MethodRef ConsoleWrite = new MethodRef ("System.Console",  "WriteLine");
			Literal  HelloWorld = "Hello World from Mr. ZADCO";


			declClass(TypePublic, "Class1");
			declEntry();
			enter (ex (ConsoleWrite, HelloWorld));

		}
	}
}
